src/iuse.cpp

```diff
@@ -8083,6 +8085,324 @@ std::optional<int> iuse::capture_monster_act( Character *p, item *it, const trip
     return 0;
 }
 
+heating_requirements heating_requirements_for_weight( const units::mass &frozen,
+        const units::mass &nfrozen )
+{
+    int ammo = 2 * divide_round_up( frozen, 500_gram ) + divide_round_up( nfrozen, 500_gram );
```

Comment:
Not sure this PR needs to tackle this, but the energy requirements for this are definitely wrong. @anoobindisguise may have some advice there.